### ISSUE TITLE
h2-build - createUserMarkup - markupArray different between Windows and Docker-Ubuntu

### ISSUE THREAD

**Issue by petertgiles** [hydrogen-design-system/hydrogen.css#37]
While doing testing with Docker I found that the h2-build command works differently between my native Windows installation and Ubuntu running in Docker.  When running h2-build for one of the application subprojects in my Ubunutu Docker container the [src(markupArray)](https://github.com/hydrogen-design-system/hydrogen.css/blob/e9b8d52927f5c11543b3e3aaaa7eb069f88d5367/lib/scripts/functions/set-markup.js#L177) function on lib/scripts/functions/set-markup.js:177 includes my common/src/css/hydrogen.css file but if I run it in my native Windows environment the common hydrogen.css file is not included.  I found this by adding logging to the pipe expression as shown at [https://www.npmjs.com/package/vinyl-fs#usage](https://www.npmjs.com/package/vinyl-fs#usage).

Node version in both environments - v14.18.1

I'm not clear on which behaviour is correct, though. 😊

Theories: maybe it is a difference with how node handles symlinks or permissions or case sensitivity between the two?

In summary:
Docker\Ubuntu -> hydrogen.css file from common is included in the markup.txt file.
Native\Windows -> hydrogen.css file from common is NOT included in the markup.txt file.

**Comment by petertgiles** [comment]
Confirmed with @JoshBeveridge that h2-build should not be trying process another hydrogen.css file.  So the Windows behaviour is correct and the Ubuntu behaviour is incorrect.

**Comment by JoshBeveridge** [comment]
Very possible that this has been addressed by #35, but this issue will stay open until we've done further testing.

**Comment by petertgiles** [comment]
Solved by version 1.1.12.  Thanks, Josh! 🎉
